src/SelectMulti.vue

```diff
@@ -273,6 +281,7 @@
 				@keydown="onInputKeyDown"
 			/>
 
+			<span role="alert" class="sr-only" v-if="notificationMessage">{{ notificationMessage }}</span>
```

Comment:
Thanks @RED143! So I think my vote is still to stick w/ `aria-live`, because I *think* both of these notification types (item removed, and no results found) are *possible* to live without. i.e. even if there are some screen reader scenarios where `role="alert"` works slightly more reliably than `aria-live`, I think a user will be OK without them.

However, when they work, I like that they won't be prefixed with "Alert:" when we use `aria-live`, because with `removed`, for example, it'll fit into the flow of info just as well as the `selected` or `checked` / `unchecked`, for other form elements. It's certainly no more important, for example, than the `selected` event, which doesn't get the `Alert:` prefix.

this isn't a strong preference on my part, though - will leave it up to you, @RED143, if you prefer `alert`, and/or @EvanSanderson